pytest_lab/locker.py

```diff
@@ -122,22 +122,31 @@ def __init__(self, config, backend, ttl=30):
         self._thread = None
         self._stop = threading.Event()
 
-    def aquire(self, key, user=None):
+    def aquire(self, key, wait_on_locks, user=None):
```

Comment:
Something I forgot to mention the first time around - don't change the public API here. Its a mater of global policy. This should be set in the constructor once when the plugin is initialised.